app/views/suggestions/index.erb

```diff
@@ -0,0 +1,32 @@
+<!-- if own_suggestion?(suggestion) working in controller. Possibly refactor below logic to not include if/else logic-->
+<div class="container">
+
+  <h2>Welcome, <%= current_user.username %>!</h2>
+
+  <h3><a href='/suggestions/new'>New Suggestion</a></h3>
+
+  <% @suggestions.each do |s| %>
+
+  <% if own_suggestion?(s) %>
```

Comment:
You actually don't need a method to check. Instead of looping through all of the suggestions, you could simply fetch the suggestions that belongs to the user and loop through that.